### DISCUSSION TITLE
Fraudulent site posing as Rectangle's site finally suspended (Thanks, all!)

### DISCUSSION THREAD

**Discussion by rxhanson** [rxhanson/Rectangle#969] [General]
UPDATE: Fraudulent site finally suspended by the registrar.

----Original Post----
This year, a fraudulent site launched that impersonates the site for Rectangle and Rectangle Pro. My official site is https://rectangleapp.com. The fraudulent site is rectanglemac(dot)com, and it sells a cracked version of Rectangle Pro. 

It's been around for several months, and has slowly picked up enough traction to where users are falling for the scam. Here are the details:

The fraudulent site is geared toward optimizing SEO with a lot of keyword-laden text. There are also Twitter, Facebook, and YouTube accounts associated with it and random sites linking to it. The site has a PayPal button, and if a user pays $5, then they are redirected to a MEGA download link where they can download the cracked version of Rectangle Pro. Cracked software is not notarized and needs you to bypass Gatekeeper to run it. 

The most unfortunate part of this for me is that it steals my time away from working on Rectangle & Rectangle Pro, which means that this scam is directly stealing from you, too. 

I need your help to at least report the fraudulent site, so that there is some potential that one of these places will help remove it from their services. Here are the links to all the places to report:

Cloudflare
https://abuse.cloudflare.com/

Google safe browsing
https://safebrowsing.google.com/safebrowsing/report_badware/
https://safebrowsing.google.com/safebrowsing/report_phish/

Microsoft Security Intelligence
https://www.microsoft.com/en-us/wdsi/support/report-unsafe-site-guest

NameCheap (the registrar)
https://support.namecheap.com/index.php?/Tickets/Submit

DuckDuckGo
Send an email to [abuse@duckduckgo.com](mailto:abuse@duckduckgo.com).

Twitter: fraudulent handle is @Rectanglemac
https://help.twitter.com/en/forms/authenticity/impersonation

If you fell for the scam and paid via PayPal, please report the fraud and get your money back.

This could end up as a game of whack a mole, but maybe with enough deterrence something positive can come of it. If you think of anything else that can be done, let me know.

**Comment by dukeofharen** [comment]
The site seems to be down at least. I reported the YouTube channel and Twitter account, so let's hope the social media account are banned quickly :)

Also, thanks for making Rectangle, have been using it for a few weeks now (just got a Mac for work) and am really happy using it!

**Reply by rxhanson** [reply]
Thanks! This is great news. I'm so relieved to not have this stuck at the forefront of my mind.
